languages/java/exercises/concept/constructors/src/test/java/RemoteControlCarTest.java

```diff
@@ -0,0 +1,125 @@
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+import org.junit.Ignore;
+
+public class RemoteControlCarTest {
```

Comment:
There are now two test files, which would mean that the in-browser editor would have to support this which I don't think it does. @iHiD are tracks allowed to have multiple test files?

Reply:
I have a recollection of splitting them because of a test runner issues with having two classes in the same file but I don't remember the details. I don't mind merging them again if it makes things easier.

Reply:
> are tracks allowed to have multiple test files?


I don't think so right now, no. If merging them is hard we can rediscuss but it's not something we've planned for anywhere.

Reply:
Cool. I'll try to merge them again and let you know if I run into problems.